static/js/components/text_search_bar.tsx

```diff
@@ -51,12 +51,25 @@ export function TextSearchBar(props: TextSearchBarPropType): JSX.Element {
             setInvalid(false);
           }}
           onKeyDown={(e) => e.key === "Enter" && handleSearch()}
-          className="pac-target-input"
+          className="pac-target-input search-input-text"
           autoFocus={props.shouldAutoFocus}
-        />
-        <Button onClick={handleSearch} id="rich-search-button">
-          <span className="material-icons search rich-search-icon">search</span>
-        </Button>
+          autoComplete="off"
+        ></Input>
+        <div onClick={handleSearch} id="rich-search-button">
```

Comment:
nit: add some comment on the source of this?

Reply:
What do you mean by "this"?

Reply:
I mean, this blob of SVG, where did you come up with its content?

Reply:
Ah, took that from the figma mocks from Ido and Adriana, should I still add a note?

Reply:
ok, nm

Reply:
I'd store this as `search-arrow.svg` in the images folder and include it here with an `<img src="/path/to.svg" />`

Reply:
Ok, will send followup PR to cleanup the svg!